### PR TITLE
REFACTOR remove Silverstripe Collection module

### PR DESCRIPTION
**Pull request by jsirish**

### CODE REVIEW

File: src/Page/JobCollection.php [hunk 1]
```diff
@@ -76,6 +76,11 @@ class JobCollection extends \Page
         Job::class,
     ];
 
+    /**
+     * @var string
+     */
+    private static $page_size = 10;
```
Comment (muskie9) [inline src/Page/JobCollection.php]: Should we move this to the CMS?

Comment (jsirish) [reply src/Page/JobCollection.php]: Yeah good idea, I'll set default to 10

**Comment by muskie9** [comment]
I can't recall, is the default layout "card" like, or is it more "feature" like? If Cards then I'd maybe do a multiple of 3 or 4 for the default. If it's like features or a list then 10 would work.

**Comment by jsirish** [comment]
more feature-like - or Blog holder with a list

**Review by muskie9 — APPROVED** [review]

**Pull request merged**